client/js/components/wobbly-edge.js

```diff
@@ -11,9 +11,14 @@ const wobblyEdge = (el) => {
     options.numberOfPoints = el.getAttribute('data-number-of-points');
   }
 
+  if (el.getAttribute('data-is-static') === 'true') {
+    options.isStatic = true;
+  }
+
   const defaults = {
     maxIntensity: 50,
-    numberOfPoints: 5
+    numberOfPoints: 5,
+    isStatic: false
   };
```

Comment:
Could we not have something like:
```JS
const defaults = {
maxIntensity: 50,
numberOfPoints: 5,
isStatic: el.getAttribute('data-is-static') === 'true'
}
```

To avoid mutation?